sklearn/metrics/classification.py

```diff
@@ -808,8 +808,8 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
            Godbole, Sunita Sarawagi
            <http://www.godbole.net/shantanu/pubs/multilabelsvm-pakdd04.pdf>`
 
-    Examples
-    --------
+    First example
```

Comment:
The former form of the heading is recognised by our documentation to HTML processor.